pkg/ruler/rules/store.go

```diff
@@ -70,7 +73,7 @@ func NewConfigRuleStore(c client.Client) *ConfigRuleStore {
 }
 
 func (c *ConfigRuleStore) ListAllUsers(ctx context.Context) ([]string, error) {
```

Comment:
Once this PR is merged (to keep diff easier), I would sugget to move `ConfigRuleStore` to a dedicated package `pkg/ruler/rules/configdb/` and keep in this file only the generic `type` definitions.